_posts/guides/2021-12-21-guide_gamemode-info-surf.md

```diff
@@ -17,7 +17,7 @@ toc_sticky: true
 
 ## What is Surf?
 
-Surf is a very popular gamemode
+Surf is a very popular gamemode that derives from the same source bug as bhop and ahop. Even thought it came from a bug it became one of the most popular minigames on games like cs:s cs:go team fortress 2 etc.
```

Comment:
I'm not sure it's right to call surf movement a "bug" (and it's certainly not the same as ahop), it's a product of increasing sv_airaccelerate significantly to give you greater air control, the same as with bhop and I believe most climb variants. I also wouldn't call it a minigame, I think gamemode/sub-game/custom server might be better.

Typo in "Even thought".

Let's try and improve our punctuation a little, "Even thought it came from a bug it became" can be "Even thought it came from a bug, it became" (though as above, I don't think it's right to call it a bug).

The game names should be capitalised.

Reply:
@Jedno last change is a little better, but please capitalise the game names properly.

Also, lets add a little bit to explain why air control is important, perhaps: _increasing sv_airaccelerate setting, giving better air control thereby allowing the player to perform sharp turns in the air. This also allows the player to "surf" along angled surfaces known as "ramps"._